packages/discord.js/src/structures/GuildAuditLogs.js

```diff
@@ -55,6 +57,18 @@ class GuildAuditLogs {
       }
     }
 
+    /**
+     * Cached application commands, includes application commands from other applications
+     * @type {Collection<Snowflake, ApplicationCommand>}
+     * @private
+     */
+    this.applicationCommands = new Collection();
+    if (data.application_commands) {
+      for (const command of data.application_commands) {
+        this.applicationCommands.set(command.id, new ApplicationCommand(guild.client, command, guild));
+      }
+    }
```

Comment:
This appears to not be documented, no?
https://discord.com/developers/docs/resources/audit-log#audit-log-object-audit-log-structure

Reply:
It supposedly was but appears to not actually be. I'll be leaving it here.